app/events/[id]/page.tsx

```diff
@@ -59,7 +61,16 @@ const EventPage = ({ params }: Props) => {
                 <span className="text-muted-foreground">Venue: </span>
                 {event.venue}
               </h2>
-              <p className="">{event.desc}</p>
+              <div className="prose lg:prose-l">
```

Comment:
make the headings a little bit smaller as it is breaking the hierarchy of the design.
![image](https://github.com/adityanandanx/tdh-ems/assets/61308761/b82a1977-dc7d-4342-b0d5-e2fac75c2791)
In this image, `Event 2` text should appear a little larger than `this is a heading` text.
`prose-md` or `prose-sm` should do the trick.

I think `prose-l` is not a valid class